src/components/Articles.jsx

```diff
@@ -31,7 +30,12 @@ const Articles = () => {
     <div className="body">
       <h1>Articles</h1>
       {articles.map((article) => {
-         return <ArticlesCard article={article} key={article.article_id} />
+        return (
+        <div key={article.article_id}>
+          <ArticlesCard article={article} /> 
+         <Link to={`/articles/${article.article_id}`}>Continue reading...</Link>
```

Comment:
Great use of the Link element to link to your ID using a parametric endpoint ⭐ 
However this Link's a little out the way and hard to click - it would make the page a bit more accessible (in particular for those on mobile) to provide a bit of a bigger target for the link, could you wrap it around more of the ArticleCard?